### PR TITLE
Issue #3500 : update jwt sso config

### PR DESCRIPTION
**Pull request by arunavemulapalli**
Update jwtsso configuration to use mpJwt instead of jwt (consumer ref) to process the token.

### CODE REVIEW

**Comment by arunavemulapalli** [comment]
fixes #3500

File: dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -20,21 +20,22 @@ jwtSso.desc=Information about configuring JWT Single Sign On.
...
 
 includeLtpaCookie=Include LTPA cookie
-includeLtpaCookie.desc=After successful authentication with a JWT token, include an LTPA cookie in addition to the SSO cookie.
+includeLtpaCookie.desc=After successful authentication with a JWT token, include an LTPA cookie in addition to the JWT cookie.
 
-fallbackToLtpa=Fall back to LTPA cookie
-fallbackToLtpa.desc=If the SSO cookie is missing or invalid, attempt to process an LTPA cookie.
+#initially this property was exposed as fallbackToLtpa but now is useLtpaIfJwtAbsent
+fallbackToLtpa=Use LTPA cookie if JWT cookie is not present
+fallbackToLtpa.desc=If the JWT cookie is missing, attempt to process an LTPA cookie if present.
```
[the remark this answers is not in the document]
Comment (arunavemulapalli) [reply dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties]: done.

File: dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -20,21 +20,22 @@ jwtSso.desc=Information about configuring JWT Single Sign On.
...
 
-setCookieSecureFlag=Set secure flag on SSO cookie
-setCookieSecureFlag.desc=Set the secure flag on the SSO cookie to send it only over a secured connection. 
+setCookieSecureFlag=Set secure flag on JWT cookie
+setCookieSecureFlag.desc=Set the secure flag on the JWT cookie to send it only over a secured connection. 
 
-jwtConsumerRef=JWT consumer reference
-jwtConsumerRef.desc=A reference to the JWT Consumer configuration element in server.xml that describes how to consume the JWT token.
+mpjwtConsumerRef=MicroProfile JWT reference
+mpjwtConsumerRef.desc=A reference to the MicroProfile JWT token configuration element in server.xml that describes how to process the JWT token.
```
[the remark this answers is not in the document]
Comment (arunavemulapalli) [reply dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties]: done.

**Comment by arunavemulapalli** [comment]
#build

File: dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties [hunk 3]
```diff
@@ -20,21 +20,22 @@ jwtSso.desc=Information about configuring JWT Single Sign On.
...
 
-setCookieSecureFlag=Set secure flag on SSO cookie
-setCookieSecureFlag.desc=Set the secure flag on the SSO cookie to send it only over a secured connection. 
+setCookieSecureFlag=Set secure flag on JWT cookie
+setCookieSecureFlag.desc=Set the secure flag on the JWT cookie to send it only over a secured connection. 
 
-jwtConsumerRef=JWT consumer reference
-jwtConsumerRef.desc=A reference to the JWT Consumer configuration element in server.xml that describes how to consume the JWT token.
+mpjwtConsumerRef=MicroProfile JWT reference
+mpjwtConsumerRef.desc=A reference to the MicroProfile JWT token configuration element in the server.xml that describes how to process the JWT token.
```
Comment (sishida) [inline dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties]: Please add the word "file" after "server.xml." These extra descriptive words can be helpful in translation. :) Thank you!

Comment (arunavemulapalli) [reply dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties]: We ended up removing this attribute from the configuration. I updated properties file to reflect this change.
Thanks.

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.jwt_fat.mpjwt

**Comment by arunavemulapalli** [comment]
#build

**Comment by arunavemulapalli** [comment]
The latest personal build has failures in other components - 
All jwtsso and related security fats are running good.

**Comment by arunavemulapalli** [comment]
#build

**Comment by arunavemulapalli** [comment]
#build

**Comment by ayoho** [comment]
Need to follow up to see if this has been merged by one of Ut's PRs.

**Pull request closed without merging**